### ISSUE TITLE
"ASCII serialization"

### ISSUE THREAD

**Issue by annevk** [WICG/feature-policy#115]
It doesn't seem like this section defines a serialization. It defines a header syntax, but not a serialization of some data structure. (I suppose this might get solved to some extent if the shared data model with CSP is sorted, but seemed worth reporting separately.)

**Comment by clelland** [comment]
You're right -- I've called them "serialized feature policy" and "serialized policy directive", but they're never really serialized *from* anything. They're only ever parsed from that representation into the internal data structures.

(CSP appears to do the same thing: I may have been inspired by that spec when I wrote it)

**Comment by hober** [comment]
(Hi. Got here in a convoluted manner from w3ctag/design-reviews#369.)

It's also weird to call this "ASCII" in the first place, isn't it? I would have expected this to look more like the [Infra spec](https://infra.spec.whatwg.org/)'s [serialize JSON to bytes](https://infra.spec.whatwg.org/#serialize-json-to-bytes)

**Comment by clelland** [comment]
Well, it's definitely not a serialization algorithm; the section just defines the textual representation of policy directives, for use in headers and the `allow` attribute. Maybe the thing to do here is to rename this to something like "text representation", and make it part of the delivery section. Then we'd be rid of the problematic heading that suggests that we're going to serialize something.

Would that make sense to you?

**Comment by annevk** [comment]
"Syntax" seems fine? But yeah, that makes sense to me generally.
